.env

```diff
@@ -92,12 +92,7 @@ DEVTOOLSET_VERSION=
 # Used through docker-compose.yml and serves as the default version for the
 # ci/scripts/install_vcpkg.sh script. Prefer to use short SHAs to keep the
 # docker tags more readable.
-#
-# Please also update the crossbow configuration in order to keep the github
-# actions cache up to date for the macOS wheels:
-#   https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml
-# TODO: Update to an official release tag https://github.com/apache/arrow/issues/35099
```

Comment:
Removed due to suggestion here: https://github.com/apache/arrow/pull/34818#discussion_r1184321238

Reply:
Thanks!